pom.xml

```diff
@@ -67,14 +69,31 @@
         <dependency>
             <groupId>org.jvnet.hudson.plugins</groupId>
             <artifactId>analysis-core</artifactId>
-            <version>1.71</version>
-            <type>hpi</type>
+            <version>1.73-beta-SNAPSHOT</version><!-- TODO: change to 1.73 before release -->
```

Comment:
I believe 1.73 has been released (see https://github.com/jenkinsci/analysis-core-plugin/tree/analysis-core-1.73). 

Can I change it to 1.73 after merging? 

ps: sorry, it took me so long to review, that actually 1.74 is out as well.